File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/LichHenResponse.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace QuanLyPhongKham.DTOs.Responses
{
    public class LichHenResponse
    {
        public Guid MaLH { get; set; }
        public DateTime ThoiGianKham { get; set; }
        public string TrangThai { get; set; }

        public Guid? MaBN { get; set; }
        public Guid? MaBS { get; set; }
        public string? CreatedBy { get; set; }
    }
}

File: QuanLyPhongKham.Client/src/interfaces/responses/LichHenResponse.js
/**
 * @typedef {Object} LichHenResponse
 * @property {string} maLH
 * @property {string|Date} thoiGianKham
 * @property {string} trangThai
 * @property {string} maBN
 * @property {string} maBS
 */

export const createLichHenResponse = (
  maLH = "",
  thoiGianKham = null,
  trangThai = "",
  maBN = null,
  maBS = null,
) => ({
  maLH,
  thoiGianKham,
  trangThai,
  maBN,
  maBS,
});
